lib/src/reporting_query_parser.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:convert';
```

Comment:
Unit test for this file will be added once I get back some reviews about the way I currently use to parse revenue from response. I don't like how I currently handle it, and I'm looking forward to any suggestions.

Reply:
I think its pretty good overall

Reply:
I changed this to using the csv package now.